### PR TITLE
added simple installation guide

### PR DESCRIPTION
**Pull request by mansimas**

### CODE REVIEW

File: Resources/doc/guide.rst [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+        managers:
+            default:
+                connection: default
+                mappings:
+                    - AcmeDemoBundle
+
+Creating Database for ElasticSearch
+-----------------------------------
+
+Go to ``src/Acme/DemoBundle`` and create new Folder ``Document``. It must be named as "Document". ONGR points to exactly this directory.
```
Comment (linasmo) [inline Resources/doc/guide.rst]: Its not entirely true. Document directory can be specified in config.
https://github.com/ongr-io/ElasticsearchBundle/blob/master/DependencyInjection/Configuration.php#L33

Comment (mansimas) [inline Resources/doc/guide.rst]: I plan to include this later, how to change Document's folder.  I could edit this line like "ONGR points to exactly this directory as default".

File: Resources/doc/guide.rst [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+.. code:: bash
+
+    mkdir Sites
+    cd Sites
+
+Create new Symfony project
+
+.. code:: bash
+
+    symfony new project
```
Comment (martiis) [inline Resources/doc/guide.rst]: This `symfony` command is available by using https://github.com/symfony/symfony-installer. It is recommended way but also mention composer `composer create-project symfony/framework-standard-edition`

**Comment by martiis** [comment]
Else looks nice :+1:

**Pull request merged**